v1/linkml-schemas/vectorsOfTrust.yaml

```diff
@@ -21,9 +21,12 @@ enums:
       P2:
       P3:
       P4:
+      PCL200:
+      PCL250:
 
 slots:
   vot:
+    description: The `vot` identifies the Vector Of Trust.
```

Comment:
Sorry to be petty but could this be rearranged to be:

```
range: IdentityVectorOfTrust
description: The `vot` identifies the Vector Of Trust.
```

rather than:

```
description: The `vot` identifies the Vector Of Trust.
range: IdentityVectorOfTrust
```

just to be consistent with other parts of this file and other files :)

Reply:
@michaelabenyohai Done